### PR TITLE
Factor isNotFromAnExternalOrGTMUser out of githubEventHelpers

### PR DESCRIPTION
**Pull request by chadwhitacre**
Part of #482, after #532.

I want to see if I can put the GraphQL wrappers on GitHubOrg. Gotta get this helper out of the way first.

### CODE REVIEW

File: src/brain/issueLabelHandler/followups.ts [hunk 1]
```diff
@@ -18,12 +18,12 @@ import {
 } from '@utils/businessHours';
 import {
   addIssueToGlobalIssuesProject,
-  isNotFromAnExternalOrGTMUser,
   modifyDueByDate,
   modifyProjectIssueField,
   shouldSkip,
 } from '@utils/githubEventHelpers';
 import { isFromABot } from '@utils/isFromABot';
+import { isNotFromAnExternalOrGTMUser } from '@utils/isNotFromAnExternalOrGTMUser';
```
Comment (chadwhitacre) [inline src/brain/issueLabelHandler/followups.ts:26]: Note parallel with `isFromABot` on the previous line.

File: src/brain/issueLabelHandler/triage.ts [hunk 2]
```diff
@@ -9,12 +9,12 @@ import {
 import { ClientType } from '@api/github/clientType';
 import { getClient } from '@api/github/getClient';
 import {
-  isNotFromAnExternalOrGTMUser,
+  addIssueToGlobalIssuesProject,
   modifyProjectIssueField,
   shouldSkip,
 } from '@utils/githubEventHelpers';
-import { addIssueToGlobalIssuesProject } from '@utils/githubEventHelpers';
```
Comment (hubertdeng123) [inline src/brain/issueLabelHandler/triage.ts:16]: 😃

**Review by hubertdeng123 — APPROVED** [review]
lgtm, but maybe should change name of PR from isNotFromBlahBlah to isNotFromAnExternalOrGTMUser?

**Comment by chadwhitacre** [comment]
> maybe should change name of PR from isNotFromBlahBlah to isNotFromAnExternalOrGTMUser

Boring. :P

**Pull request merged**